lib/config/configuration.go

```diff
@@ -1970,10 +1970,20 @@ func applyWindowsDesktopConfig(fc *FileConfig, cfg *servicecfg.Config) error {
 	if err != nil {
 		return trace.Wrap(err)
 	}
+	if len(cfg.WindowsDesktop.Hosts) > 0 {
+		log.Warnln("hosts field is deprecated, use static_hosts instead")
```

Comment:
```suggestion
		log.Warnln("hosts field is deprecated, prefer static_hosts instead")
```

I think this might make it a little more clear that `hosts` still works, but `static_hosts` should be preferred.